xclim/indicators/atmos/_temperature.py

```diff
@@ -437,6 +438,18 @@ class TasminTasmax(Daily2D):
     compute=indices.cold_spell_frequency,
 )
 
+cool_night_index = Tasmin(
+    identifier="cool_night_index",
+    units="degC",
+    long_name="cool night index",
+    description="Mean minimum temperature for September (northern hemisphere) or March (southern hemisphere).",
+    cell_methods="time: min within days time: mean over days",
+    comment="Original formula published in Tonietto & Carbonneau, 2004 (10.1016/j.agrformet.2003.06.001).",
+    abstract="A night coolness variable which takes into account the mean minimum night temperatures during the "
+    "month when ripening usually occurs beyond the ripening period.",
+    allowed_periods=["A"],
+    compute=wrapped_partial(indices.cool_night_index, suggested=dict(lat=_empty)),
```

Comment:
Is there a reason to manually _force_ `lat` here ?  En fait, the indice is currently broken I think, if you didn't pass `lat`. Even if a lot of use cases would be in only one hemisphere, I think the simplest take on our side would be to make it non-optional directly in the indice.

Reply:
Yes, true. It needs to be there (IE: we can't compute without it). I can adjust this.